docs/tutorials/07_AE_workflow.ipynb

```diff
@@ -2376,474 +291,100 @@
-      "text/plain": [
-       "<Figure size 500x400 with 1 Axes>"
-      ]
-     },
-     "execution_count": 15,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
     "em.plot_eval(gp_final)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 9 - Sensitivity Analysis \n",
+    "Use AutoEmulate to perform sensitivity analysis. This will help identify the parameters that have higher impact on the outputs to narrow down the search space forperforming model calibration. \n",
```

Comment:
```suggestion
    "Use AutoEmulate to perform sensitivity analysis. This will help identify the parameters that have higher impact on the outputs to narrow down the search space for performing model calibration. \n",
```